src/documents/consumer.py

```diff
@@ -191,6 +200,9 @@ def _ocr(self, pngs, lang):
         Performs a single OCR attempt.
         """
 
+        if not pngs:
```

Comment:
I see what's going on now, thanks.

As for adding a check at the top of `_get_ocr()`, I suppose it'd be good accounting, even though that *should* never happen ("should", famous last words).  Alright, if you add a check at the top as you suggest, I'll merge this bit happily.